presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergRecordFileWriter.java

```diff
@@ -21,15 +21,20 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.hadoop.mapred.JobConf;
 import org.apache.iceberg.Metrics;
+import org.apache.iceberg.MetricsConfig;
+import org.apache.iceberg.hadoop.HadoopInputFile;
+import org.apache.iceberg.parquet.ParquetUtil;
 
 import java.util.List;
-import java.util.Optional;
 import java.util.Properties;
 
 public class IcebergRecordFileWriter
```

Comment:
This class is now unused and can be deleted

Reply:
That's nice!  Deleted.